include/databaseFunctions_blog_pic.php

```diff
@@ -152,19 +352,24 @@ function DeletePost($postID){
 			WHERE postID = $postID
 		")->fetch();
 		echo"Post Deleted.<br>";
+		ResetAuto(GetTotalPosts());
 	}
 
-	function isPicture($postID){
-		if(GetPost($postID)['isPic']==1){
-			return true;
-		}
-		return false;
+	function GetPostType($postID){
+		return GetPost($postID)['postType'];
 	}
 
-	function getTotalPosts(){
+	function GetTotalPosts(){
 		$result = dbQuery("
-			SELECT COUNT(postID) AS count
+			SELECT MAX(postID) AS count
 			FROM posts
 		")->fetch();
 		return $result['count'];
 	}
+
+	function ResetAuto($count){
```

Comment:
This function seems like a bad idea to me. The primary key shouldn't mean anything, and you shouldn't care what it is. If you're using it to mean something (like counting the posts) you should just use a different field for that.